components/brave_new_tab_ui/components/default/braveToday/customize/FeedCard.tsx

```diff
@@ -9,10 +9,17 @@ import { getCardColor } from './colors'
 import { useGetUnpaddedImage } from '../cards/CardImage'
 import { api } from '../../../../api/brave_news/news'
 
-const Card = styled('div') <{ backgroundColor?: string }>`
+interface CardProps {
+  backgroundColor?: string
+}
+
+const Card = styled('div').attrs<CardProps>(props => ({
```

Comment:
:+1: On the one hand, it's interesting that this has such an impact on performance, but I guess it makes sense if behind the scenes styled components is constructing, deduping and attaching stylesheets.